src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/Models/Manifest.cs

```diff
@@ -103,4 +106,69 @@ public class Blob
         [XmlAttribute(AttributeName = "NonShipping")]
         public bool NonShipping { get; set; }
     }
+
+    [XmlRoot(ElementName = "SigningInformation")]
+    public class SigningInformation
+    {
+        [XmlAttribute(AttributeName = "AzureDevOpsCollectionUri")]
+        public string AzureDevOpsCollectionUri { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsProject")]
+        public string AzureDevOpsProject { get; set; }
+
+        [XmlAttribute(AttributeName = "AzureDevOpsBuildId")]
+        public string AzureDevOpsBuildId { get; set; }
+
+        [XmlElement(ElementName = "FileExtensionSignInfo")]
+        public List<FileExtensionSignInfo> FileExtensionSignInfos { get; set; }
+
+        [XmlElement(ElementName = "FileSignInfo")]
+        public List<FileSignInfo> FileSignInfos { get; set; }
+
+        [XmlElement(ElementName = "ItemsToSign")]
+        public List<ItemsToSign> ItemsToSign { get; set; }
+
+        [XmlElement(ElementName = "StrongNameSignInfo")]
+        public List<StrongNameSignInfo> StrongNameSignInfos { get; set; }
+    }
+
+    [XmlRoot(ElementName = "FileExtensionSignInfo")]
+    public class FileExtensionSignInfo
+    {
+        [XmlAttribute(AttributeName = "Extension")]
+        public string Extension { get; set; }
+
+        [XmlAttribute(AttributeName = "CertificateName")]
+        public string CertificateName { get; set; }
+    }
+
+    [XmlRoot(ElementName = "FileSignInfo")]
+    public class FileSignInfo
+    {
+        [XmlAttribute(AttributeName = "File")]
+        public string File { get; set; }
+
+        [XmlAttribute(AttributeName = "CertificateName")]
+        public string CertificateName { get; set; }
+    }
+
+    [XmlRoot(ElementName = "ItemsToSign")]
+    public class ItemsToSign
```

Comment:
[nit] This seems to be a single ItemToSign rather than Items

Reply:
That's the name of the element i.e. `<ItemsToSign File="Microsoft.DotNet.Build.Tasks.Feed.5.0.0-ci.nupkg" />`. @jonfortescue fyi

Reply:
I'm with @riarenas on this. It seems to only represent a single item to sign.

Reply:
Ultimately this is based on the [Build Manifest (signing)](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/src/GenerateAzureDevOpsBuildManifest.cs#L15) that @chcosta made in the Arcade SDK. This model is in parity with the [SigningInformationModel](https://github.com/dotnet/arcade/pull/5517/files#diff-a3a076be8faf428780c5bb3c62d363c3R17) introduced in my PR.

I can change it in all places in my PR but that's where this ultimately derives from.

Reply:
[`sign.proj`](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Sign.proj#L46) expects `ItemsToSign` so I assume that's why we went this naming even though we are including a single item. Like this, we can just use the metadata as is without having to do any change